### PR TITLE
Update Rococo chainspec to v2_1

### PR DESCRIPTION
**Pull request by PierreBesson**

### CODE REVIEW

**Review by rphmeier — APPROVED** [review]

File: node/service/src/chain_spec.rs [hunk 1]
```diff
@@ -778,8 +778,8 @@ fn rococo_staging_testnet_config_genesis(wasm_binary: &[u8]) -> rococo_runtime::
 
 	// subkey inspect "$SECRET"
 	let endowed_accounts = vec![
-		// 5FeyRQmjtdHoPH56ASFW76AJEP1yaQC1K9aEMvJTF9nzt9S9
-		hex!["9ed7705e3c7da027ba0583a22a3212042f7e715d3c168ba14f1424e2bc111d00"].into(),
+		// 5DwBmEFPXRESyEam5SsQF1zbWSCn2kCjyLW51hJHXe9vW4xs
+		hex!["0x52bc71c1eca5353749542dfdf0af97bf764f9c2f44e860cd485f1cd86400f649"].into(),
```
Comment (rphmeier) [inline node/service/src/chain_spec.rs:782]:
```suggestion
		hex!["52bc71c1eca5353749542dfdf0af97bf764f9c2f44e860cd485f1cd86400f649"].into(),
```

**Comment by PierreBesson** [comment]
Note: parachain which want to connect the the new Rococo, will have to run with `-- --chain new-rococo-chainspec.json` until an update is released that include the new Rococo chainspec.

**Comment by tomaka** [comment]
Can we not merge this? Is anything blocking?

**Comment by NukeManDan** [comment]
Can this be backported to 0.9.18?

**Pull request merged**